src/utils.ts

```diff
@@ -0,0 +1,164 @@
+import * as Protobuf from 'protobufjs';
```

Comment:
Could these new classes be moved to `src/entity.ts`?

Reply:
ok

Reply:
@stephenplusplus  Moved into entity.ts